Add note to 'Choose a partition key' section. (#95004)
Azure/azure-content: articles/cosmos-db/partitioning-overview.md

If you need [multi-item ACID transactions](database-transactions-optimistic-concurrency.md#multi-item-transactions) in Azure Cosmos DB, you will need to use [stored procedures or triggers](how-to-write-stored-procedures-triggers-udfs.md#stored-procedures). All JavaScript-based stored procedures and triggers are scoped to a single logical partition.

> > [!NOTE]
> > If you only have one physical partition, the value of the partition key may not be relevant as all queries will target the same physical partition. 

Reviewer: ```suggestion
> [!NOTE]
> If you only have one physical partition, the value of the partition key may not be relevant as all queries will target the same physical partition. 

```
adding a line break to keep the markdown clean

If you need [multi-item ACID transactions](database-transactions-optimistic-concurrency.md#multi-item-transactions) in Azure Cosmos DB, you will need to use [stored procedures or triggers](how-to-write-stored-procedures-triggers-udfs.md#stored-procedures). All JavaScript-based stored procedures and triggers are scoped to a single logical partition.

> > [!NOTE]
> > In case you only have one physical partition, partition key may not be relevant as all queries will hit the same physical partition. 
> 

Reviewer 2: ```suggestion
> [!NOTE]
> If you only have one physical partition, the value of the partition key may not be relevant as all queries will target the same physical partition. 
```

Just some quick editing, otherwise this note is great!

Author: Hi @seesharprun, Thanks for reviewing :)  there is just a typo in the word you

Reviewer 2: Sorry! I fixed it 😄
